### ISSUE TITLE
Add filter UI components for all filter types

### ISSUE THREAD

**Issue by avaleske** [CityOfDetroit/bloom#59]
Part of CityOfDetroit/affordable-housing-app#76 

Add the UI components for all filter types. Blocked by other issues in this epic

Done: There's a UI for filtering each filter type!

TBD: This may need to be broken into smaller chunks if it's determined that we have a lot of filters to add.

**Comment by avaleske** [comment]
This is a duplicate of #102 and #103, so closing as a duplicate
